share/revng/test/configuration/revng-qa/for-runtime-abi-tests.yml

```diff
@@ -105,12 +144,37 @@ commands:
   - type: revng-qa.runtime-abi-tests
     from:
       - type: source
-        filter: for-runtime-abi-tests
+        filter: for-runtime-abi-tests and !windows
     suffix: /
     command: |-
-      "${INPUT}/build-abi-test-binaries.sh"
+      "${INPUT}/build_abi_test_binaries_with_gcc.sh"
         "${INPUT}"
         "${OUTPUT}"
         "${TRIPLE}"
-        "${CFLAGS}"
+        "${CFLAGS} ${COMMON_FLAGS}"
         "${OBJDUMP_FLAGS}"
+
+#  TODO: re-enable once new PDB support is added
+#  - type: revng-qa.runtime-abi-tests
+#    from:
+#      - type: source
+#        filter: for-runtime-abi-tests and windows
+#    suffix: /
+#    pool: wine
+#    command: |-
+#      if [[ "${QEMU_NAME}" =~ ^x86_64$$ ]]; then
+#        MSVC_TRIPLE="x86_64-winsdk-vc19-";
+#      elif [[ "${QEMU_NAME}" =~ ^i386$$ ]]; then
+#        MSVC_TRIPLE="i386-winsdk-vc19-";
+#      else
+#        echo "Unsupported architecture" > /dev/stderr;
+#        exit 1;
+#      fi;
+#      "${INPUT}/build_abi_test_binaries_with_msvc.sh"
+#        "${INPUT}"
+#        "${OUTPUT}"
+#        "${TRIPLE}"
+#        "$${MSVC_TRIPLE}"
+#        "${CFLAGS} ${COMMON_FLAGS}"
+#        "${MSVC_CFLAGS} ${COMMON_FLAGS}"
+#        "${MSVC_LDFLAGS}";
```

Comment:
1. Can we use `TRIPLE` instead of `QEMU_NAME`?
2. Can we push the computation of `MSVC_TRIPLE` into the script?
3. I'm not sure about the state of this. I don't want to merge unused code (`build_abi_test_binaries_with_msvc.sh`).
   1. Is your plan to wait for the PDB bug to be fixed before merging this?
   2. AFAIU we need newer versions of MSVC in order to test this. So how have you been working so far, given that newer PDBs are broken?
   3. Can we start to merge some stuff using only older PDB format?

Reply:
1-2. 👌

3. I really want to flush this. I have been working on this for way too long and would like to synchronize the states, so to say.

> So how have you been working so far, given that newer PDBs are broken?

Even if importing is broken, it's not hard to "emulate it": make models for interesting functions manually and then run the tests with those. For obvious reason, I only did a couple, but the artifacts look good, now we only need the non-manual importing to work for everything to click.

Either of the options work for me:
1. Remove only the `for-runtime-abi-tests.yml` part for now - reintroduce it once PDBs are supported.
2. Make this produce the artifacts (that I already verified to be good), but add a tag for revng-side to ignore them until PDBs are fixed.
3. Try to see if it would work only with `vc16`, considering we no longer need assembly, but I'm skeptical about this.
4. Try to look if there's a way to run `vc19` in some kind of compatibility mode and produce older PDBs (doubt it).

My preferred option is 2, I guess.

Reply:
Closing this thread since this code is uncommented again.